packages/coordinator/src/components/Popup/PopupFlashMessage.tsx

```diff
@@ -0,0 +1,28 @@
+import { Fade } from "@material-ui/core";
+import styles from "./Popup.module.scss";
+
+type PopupFlashMessageProps = {
+  message: string;
+  showFlash: boolean;
+  hide: () => void;
+};
+
+export default function PopupFlashMessage({
+  message,
+  showFlash,
+  hide,
+}: PopupFlashMessageProps) {
+  if (showFlash) {
+    setTimeout(() => {
+      hide();
+    }, 700);
+  }
```

Comment:
This `if` should be in a `useEffect`, this way it won't be triggered too many times. It should depend on `showFlash` and `hide`